harness/determined/pytorch/_pytorch_context.py

```diff
@@ -940,6 +942,39 @@ def get_tensorboard_path(self) -> pathlib.Path:
         """
         return self._core.train.get_tensorboard_path()
 
+    def get_tensorboard_writer(self) -> Any:
+        """
+        This function returns an instance of `torch.utils.tensorboard.SummaryWriter`
```

Comment:
Most of this code block looks like excellent sphinx syntax, but this class name will be in _italics_ rather than `monospace`.  You need double backticks before and after.